types/params_test.go

```diff
@@ -209,27 +209,27 @@ func TestConsensusParamsValidation(t *testing.T) {
 			params: makeParams(makeParamsArgs{
 				blockBytes:   -2,
 				evidenceAge:  2,
-				precision:    1,
-				messageDelay: 1,
+				precision:    time.Nanosecond,
+				messageDelay: time.Nanosecond,
 			}),
 			valid: false,
 		},
-		// test invalid pubkey type provided
+		// test invalid synchrony params
 		{
-			name: "messageDelay -1",
+			name: "messageDelay 0",
 			params: makeParams(makeParamsArgs{
 				evidenceAge:  2,
-				precision:    1,
-				messageDelay: -1,
+				precision:    time.Nanosecond,
+				messageDelay: 0,
 			}),
 			valid: false,
 		},
 		{
-			name: "precision -1",
+			name: "precision 0",
 			params: makeParams(makeParamsArgs{
 				evidenceAge:  2,
-				precision:    -1,
-				messageDelay: 1,
+				precision:    0,
```

Comment:
And this one?

Reply:
Anything <= 0 is invalid, so this is the largest value that already triggers the failure.

Reply:
0 should be invalid

Reply:
Same, let's test both, in case someone, 3 years from now, decides it should be `<0` in production